code-of-conduct.md

```diff
@@ -0,0 +1,36 @@
+Workshop Code of Conduct
+========================
+
+## Introduction
+
+All DIBSI attendees are expected to agree with the following code of
+conduct. We will enforce this code as needed. We expect cooperation
+from all attendees to help ensuring a safe environment for everybody.
+
+Please see the [Event Guidelines and Expectations](http://ivory.idyll.org/dibsi/event-expectations.html) for more information.
```

Comment:
Instead of saying "for more information" can we be more specific? For example:

Please see the Event guidelines and expectations for our goals of how to respect and interact with each other during our workshop.